cli/azd/cmd/down.go

```diff
@@ -114,21 +114,10 @@ func (a *downAction) Run(ctx context.Context) (*actions.ActionResult, error) {
 	startTime := time.Now()
 
 	destroyOptions := provisioning.NewDestroyOptions(a.flags.forceDelete, a.flags.purgeDelete)
-	destroyResult, err := infraManager.Destroy(ctx, destroyOptions)
-	if err != nil {
+	if _, err = infraManager.Destroy(ctx, destroyOptions); err != nil {
 		return nil, fmt.Errorf("deleting infrastructure: %w", err)
 	}
 
-	// Remove any outputs from the template from the environment since destroying the infrastructure
-	// invalidated them all.
-	for outputName := range destroyResult.Outputs {
```

Comment:
This logic is also present in the provisioning manager (it was likely duplicated when we first built that component) so there removal here is just removing an extra copy of the code that would be a no-op.

Reply:
ref: https://github.com/Azure/azure-dev/blob/8230e0298e2a30b8d1956268b93ed402be04ab7e/cli/azd/pkg/infra/provisioning/manager.go#L99-L118